packages/editor/src/components/provider/index.native.js

```diff
@@ -114,6 +115,12 @@ class NativeEditorProvider extends Component {
 			}
 		);
 
+		this.subscriptionRemoveAllBlocks = subscribeParentRemoveAllBlocks(
```

Comment:
You're assigning the result of `subscribeParentRemoveAllBlocks`, but it doesn't look like you're doing anything with the result anywhere. Is that intentional?

Reply:
So I did do this intentionally because I was following the example of the lines right below -- `this.subscriptionParentSetTitle`.

Reply:
Were they doing that for some reason? If so, is there anything that makes our case different that we don't need it? Or if they weren't doing it for some reason, should we remove the assignment anywhere it's not needed?